index.d.ts

```diff
@@ -155,8 +155,7 @@ declare namespace Eris {
 
   // Presence/Relationship
   type ActivityFlags = Constants["ActivityFlags"][keyof Constants["ActivityFlags"]];
-  type ActivityType = BotActivityType | Constants["ActivityTypes"]["CUSTOM"];
-  type BotActivityType = Constants["ActivityTypes"][Exclude<keyof Constants["ActivityTypes"], "CUSTOM">];
+  type ActivityType = Constants["ActivityTypes"];
```

Comment:
```suggestion
  type ActivityType = Constants["ActivityTypes"][keyof Constants["ActivityTypes"]];
```
was referring to the object itself before, not the values